### ISSUE TITLE
XSS bypassing Modsecurity WAF

### ISSUE THREAD

**Issue by devplayer55221** [coreruleset/coreruleset#3381]
### Description

I have been able to achieve XSS bypassing Modsecurity WAF rules in a particular scenario.
Using ejs template, the code is rendering the user input as the value to a variable in javascript.
I am attaching screenshots for your reference.
It seems 'confirm' is not blocked, whereas 'alert' is blocked.

### How to reproduce the misbehavior 

Render user input in the javascript variable initialization.

Actual code
![image](https://github.com/coreruleset/coreruleset/assets/29159942/5c74feb9-8f8e-4335-81b2-b87225916a20)
Alert box
![image](https://github.com/coreruleset/coreruleset/assets/29159942/a9f992ea-36e9-44e7-9177-044e7b1c796e)
Payload
![image](https://github.com/coreruleset/coreruleset/assets/29159942/f3650f95-5df2-419c-a74d-0af5388ad522)
Source code on executing request
![image](https://github.com/coreruleset/coreruleset/assets/29159942/67768387-8cba-4e05-bb46-4444c62bb2e4)

### Your Environment

* CRS version : 4.0.0-rc2
* Paranoia level setting : Default
* ModSecurity version : 3.0.10
* Web Server and version or cloud provider / CDN : Nginx 1.24.0
* Operating System and version: Kali 2023.3

### Confirmation

I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by theMiddleBlue** [comment]
Hi @devplayer55221 

thank you for your report. I can confirm that CRS is blocking `alert` and not `confirm` on rule 941390 at Paranoia Level 1. As confirmed by our sandbox:

alert:
```shell
$ curl -H 'x-crs-paranoia-level: 1' -H "x-format-output: txt-matched-rules" 'https://sandbox.coreruleset.org/?a=44;alert(1);'

941390 PL1 Javascript method detected
```

confirm:
```shell
$ curl -H 'x-crs-paranoia-level: 1' -H "x-format-output: txt-matched-rules" 'https://sandbox.coreruleset.org/?a=44;confirm(1);'
```

We'll let you know our next step as soon as possible. Thank you!

**Comment by devplayer55221** [comment]
Thank you for acknowledgement. I will be waiting for your reply.

**Comment by theseion** [comment]
Fixed by #3395
